hedera-node/hedera-app/src/main/java/com/hedera/node/app/version/HederaSoftwareVersion.java

```diff
@@ -90,6 +90,11 @@ public int compareTo(@NonNull final SoftwareVersion other) {
         }
     }
 
+    @Override
+    public SoftwareVersion buildForDeserialization() {
```

Comment:
Note: nullity annotations are not available in this module without gradle changes